src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SscsCaseData.java

```diff
@@ -595,7 +594,7 @@ public SchedulingAndListingFields getSchedulingAndListingFields() {
 
     @JsonIgnore
     public JointParty getJointParty() {
-        if (isNull(jointParty)) {
+        if (jointParty == null) {
```

Comment:
Why was this changed?

Reply:
@nickwest1 given a comment that in the previous branch, he suggest align using == null / isNull